src/multi-select/dropdown.tsx

```diff
@@ -125,6 +139,11 @@ const Dropdown = ({
   const toggleExpanded = () =>
     setExpanded(isLoading || disabled ? false : !expanded);
 
+  const handleClearSelected = (e) => {
+    e.stopPropagation();
```

Comment:
I think by default button behaves as submit on firefox
If you add type button `<input type="button">X</button>` then there will be no need of `e.stopPropagation();`

Reply:
@harshzalavadiya if we add an input tag we will not be able to add a child element to it. So I added type="button" property to <button> so it does not act as form submit. 

I think you mistook e.preventDefault for e.stopPropogation here. e.stopPropogation stops the event from bubbling to parent click events so that dropdown is not toggled on clicking this button.

Reply:
@Kannndev sorry i typed input instead of button 😶, 

Event bubbling, yes that makes sense now